FiveCalls/Podfile

```diff
@@ -10,12 +10,17 @@ end
 
 target 'FiveCalls' do
   common_pods
+  pod 'OneSignal', '>= 2.5.2', '< 3.0'
```

Comment:
`pod 'OneSignal', ~> '2.5'` is more what we want. Meaning, _give me the highest in the 2.x range_.

Reply:
👍